src/marlin_stubs/M865.cpp

```diff
@@ -51,7 +48,6 @@ void PrusaGcodeSuite::M865() {
 
     } else if (const auto ft = p.option<FilamentType>('F')) {
         filament_type = *ft;
-
```

Comment:
Please keep this line

Reply:
Commit format tool doesn't like this line :)